src/Microsoft.ML.Transforms/LambdaTransform.cs

```diff
@@ -219,32 +174,12 @@ private static ITransformer Create(IHostEnvironment env, ModelLoadContext ctx)
     /// </summary>
     internal abstract class LambdaTransformBase : ICanSaveModel
```

Comment:
>ICanSaveModel [](start = 50, length = 13)

Since this doesn't need to implement `ICanSaveModel`, and there is no other functionality in this base class, I would delete it. You mentioned that it is being used internally only as an `IDataView`, in that case I would make the deriving classes (I only found one, `StatefulFilterTransform`) implement `IDataView` instead of `ITransformTemplate`.

Reply:
Thanks @yaeldekel. I have deleted `LambdaTransformBase`. I replaced `ITransformTemplate` with `IDataTransform` instead of `IDataView` because `SequentialTransformBase` in the time series code was relying on it being `IDataTransform`

---
In reply to: [466166529](https://github.com/dotnet/machinelearning/pull/5338#discussion_r466166529) [](ancestors = 466166529)

Reply:
Never mind. Your advice was correct. The earlier fix resulted in test failures. I have now changed `StatefulFilterTransform` to implement `IDataView`

---
In reply to: [466589874](https://github.com/dotnet/machinelearning/pull/5338#discussion_r466589874) [](ancestors = 466589874,466166529)